generators/client/templates/src/main/webapp/app/components/login/_login.service.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
```

Comment:
I move all the files about login in components/login/ since it's not a page anymore but a modal window.

Should I move all the files back to account/login ?